test/Core.Test/OrganizationFeatures/Groups/CreateGroupCommandTests.cs

```diff
@@ -0,0 +1,67 @@
+﻿using Bit.Core.Entities;
+using Bit.Core.Enums;
+using Bit.Core.Models.Data;
+using Bit.Core.OrganizationFeatures.Groups;
+using Bit.Core.Repositories;
+using Bit.Core.Services;
+using Bit.Test.Common.AutoFixture;
+using Bit.Test.Common.AutoFixture.Attributes;
+using NSubstitute;
+using Xunit;
+
+namespace Bit.Core.Test.OrganizationFeatures.Groups;
+
+[SutProviderCustomize]
+public class CreateGroupCommandTests
+{
+    [Theory]
+    [BitAutoData]
+    public async Task CreateGroup_Success(SutProvider<CreateGroupCommand> sutProvider, Group group, Organization organization)
+    {
+        sutProvider.GetDependency<IGroupRepository>()
+            .GetByIdAsync(group.Id)
+            .Returns(group);
+        var utcNow = DateTime.UtcNow;
+
+        await sutProvider.Sut.CreateGroupAsync(group, organization);
+
+        await sutProvider.GetDependency<IGroupRepository>().Received(1).CreateAsync(group);
+        await sutProvider.GetDependency<IEventService>().Received(1).LogGroupEventAsync(group, Enums.EventType.Group_Created);
+        Assert.True(group.CreationDate - utcNow < TimeSpan.FromSeconds(1));
```

Comment:
> This looks flaky! It makes me think we should wrap `DateTime` in a service so that we can mock it for testing. Out of scope here though, just thought I'd flag it.

I didn't find a simple way of doing this without modifying every single usage of DateTime to use a new custom interface.

Reply:
Yes, I think that's what you'd have to do, which is why I think it's out of scope here.